.github/workflows/documentation.yml

```diff
@@ -29,9 +29,13 @@ jobs:
           python3 -m pip install https://github.com/odlgroup/odl/archive/master.zip
       - name: Run doctests
         run: |
-          find . -name '*.py' | xargs grep -l '>>>' | while read file; do
-            echo "Running doctests in $file"
-            python -m doctest -v "$file"
+          find . \( -name '*.py' -o -name '*.rst' \) | while read file; do
+              if [[ "$file" != *.datasets.rst && "$file" != *.multigpu.rst ]]; then
```

Comment:
not testing the multigpu file because if does not make sense on the cpu workflow

Reply:
ok I thought adding an if torch.cuda.cuda_available(), but it was failing due to some weird indentation issue